lockc/src/lib.rs

```diff
@@ -366,6 +366,13 @@ impl<'a> BpfContext<'a> {
         }
         link_open.pin(path_link_open)?;
 
+        let mut link_setuid = skel.progs_mut().setuid_audit().attach_lsm()?;
```

Comment:
Now you need to rebase and use Aya to load your new program here:

https://github.com/rancher-sandbox/lockc/blob/main/lockc/src/load.rs#L31-L70

Reply:
Ok, rebased :)